python/ray/data/datasource/parquet_datasource.py

```diff
@@ -56,6 +57,64 @@ def _deregister_parquet_file_fragment_serialization():
     ray.util.deregister_serializer(ParquetFileFragment)
 
 
+# easier to test when isolated from retry logic
```

Comment:
nit: Just describe what this function does? Modularized code has a natural benefit of being easier to test.

Reply:
Added function description